lib/services/local/users.go

```diff
@@ -72,6 +74,13 @@ func NewIdentityService(backend backend.Backend) *IdentityService {
 	}
 }
 
+func (s *IdentityService) Clock() clockwork.Clock {
```

Comment:
Same, could we avoid making this public?

Reply:
I think we actually want this one public. It overwrites the already exported (embedded) `s.(Backend).Clock()` method.

Reply:
I'm not sold that we want to further expose the 2nd clock, I'm on the fence about it even being worth having just for a single test.